Doc. Clarification about ArgMax/Min behavior (#50607)
ClickHouse/ClickHouse: docs/en/sql-reference/aggregate-functions/reference/argmin.md

│ worker               │
└──────────────────────┘
```

**Extended example**

```sql
CREATE TABLE test
(
    a Nullable(String),
    b Nullable(Int64)
)
ENGINE = Memory AS
SELECT *
FROM values((NULL, 0), ('a', 1), ('b', 2), ('c', 2), (NULL, NULL), ('d', NULL));

select * from test;
┌─a────┬────b─┐
│ ᴺᵁᴸᴸ │    0 │
│ a    │    1 │
│ b    │    2 │
│ c    │    2 │
│ ᴺᵁᴸᴸ │ ᴺᵁᴸᴸ │
│ d    │ ᴺᵁᴸᴸ │
└──────┴──────┘

select argMin(a, b), min(b) from test;
┌─argMin(a, b)─┬─min(b)─┐

> │ a            │      0 │ -- argMin = a because it the first not-Null value, min(b) is from another row!

Reviewer: ```suggestion
│ a            │      0 │ -- argMin = a because it the first not Null value, min(b) is from another row!
```

└──────────────┴────────┘

select argMin(tuple(a), b) from test;
┌─argMin(tuple(a), b)─┐

> │ (NULL)              │ -- Tuple allows to get Null value.

Reviewer: ```suggestion
│ (NULL)              │ -- The a `Tuple` that contains only a `NULL` value is not `NULL`, so the aggregate functions won't skip that row because of that `NULL` value
```

└─────────────────────┘

select (argMin((a, b), b) as t).1 argMinA, t.2 argMinB from test;
┌─argMinA─┬─argMinB─┐
│ ᴺᵁᴸᴸ    │       0 │ -- you can use Tuple and get both (all - tuple(*)) columns for the according max(b)
└─────────┴─────────┘

select argMin(a, b), min(b) from test where a is Null and b is Null;
┌─argMin(a, b)─┬─min(b)─┐

> │ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- Nulls are not skipped because only Null values are available

Reviewer: ```suggestion
│ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- ll aggregated rows contains at least one `NULL` value because of the filter, so all rows are skipped, therefore the result will be `NULL`
```

└──────────────┴────────┘

select argMin(a, (b, a)), min(tuple(b, a)) from test;
┌─argMin(a, tuple(b, a))─┬─min(tuple(b, a))─┐

> │ d                      │ (NULL,NULL)      │ -- 'd' is the first Not null value for the min

Reviewer: ```suggestion
│ d                      │ (NULL,NULL)      │ -- 'd' is the first not Null value for the min
```

└────────────────────────┴──────────────────┘

select argMin((a, b), (b, a)), min(tuple(b, a)) from test;
┌─argMin(tuple(a, b), tuple(b, a))─┬─min(tuple(b, a))─┐
│ (NULL,NULL)                      │ (NULL,NULL)      │
└──────────────────────────────────┴──────────────────┘

> 

Reviewer: Without further explanation I don't see a good reason to include this example.

Author: fixed

select argMin(a, tuple(b)) from test;
┌─argMax(a, tuple(b))─┐

> │ b                   │ -- Tuple can be used in `Min` to not skip Nulls in `Min`

Reviewer: ```suggestion
│ d                   │ -- Tuple can be used in `min` to not skip rows with Null values as b. 
```

> select argMin(a, tuple(b)) from test;

Reviewer: I will handle this in a different PR
